app/src/main/java/com/example/fashi_shop/model/cart/Cart.java

```diff
@@ -14,20 +14,20 @@
 
     SQLiteDatabase database;
 
-    public void ConnectSQL(Context context){
+    public void ConnectSQL(Context context) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.